src/class-parsely.php

```diff
@@ -50,6 +50,39 @@ class Parsely {
 		'parsely_wipe_metadata_cache' => false,
 	);
 
+	/**
+	 * Declare post types that Parsely will process as posts
+	 *
+	 * @see https://www.parse.ly/help/integration/jsonld#distinguishing-between-posts-and-pages
+	 * @since 2.5.0
+	 * @var string[]
+	 */
+	private $allowed_jsonld_post_types = array(
```

Comment:
Any post type is "allowed" (allowed by whom?) - what we're focusing on here is the post types that are supported / recognised by Parse.ly.

Consider updating the variable to be `$supported_jsonld_post_types` (likewise for the other variable). Avoid `$recognised_jsonld_post_types` as that changes spelling between UK and US English.